app/src/components/UI/Molecules/Form Group/AccountSettingsFormGroup.tsx

```diff
@@ -37,6 +38,7 @@ const AccountSettings: React.FC<{
   const [updateSetting] = useUpdateUserMutation();
 
   const handleSubmit = async () => {
+    dispatch(setName({ firstName: firstName, lastName: lastName }));
```

Comment:
The name state is only set if the user clicks the button in this page. The name should be set regardless of where the user is. If the user is on the landing page, the name should be visible.